docs/entity-manager-api.md

```diff
@@ -63,6 +63,11 @@ const rawData = await manager.query(
     'SELECT * FROM USERS WHERE name = @0 and age = @1',
     [ 'John', 24 ]
 )
+
+-   `sql` - Executes a raw SQL query using template literals.
+
+```typescript
+const rawData = await manager.sql`SELECT * FROM USERS WHERE name = ${'John'} and age = ${24}`
 ```
```

Comment:
Same again here.

````suggestion
```

-   `sql` - Executes a raw SQL query using template literals.

```typescript
const rawData = await manager.sql`SELECT * FROM USERS WHERE name = ${'John'} and age = ${24}`
```

Learn more about using the [SQL Tag syntax](sql-tag.md).
````